python/snippets/mov_avg.py

```diff
@@ -0,0 +1,61 @@
+import os
```

Comment:
Should we add the shebang?

#!/usr/bin/python3

Reply:
I have no clue how I review file permissions in github, but I assume that this file isn't executable since it doesn't have a shebang, should we make it executable?